sbt_common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/IdentifierType.scala

```diff
@@ -0,0 +1,21 @@
+package uk.ac.wellcome.models.work.internal
+
+import uk.ac.wellcome.models.utils.CSVReferenceTypeManager
+
+case class IdentifierType(
+  id: String,
+  label: String,
+  ontologyType: String = "IdentifierType"
+)
+
+case object IdentifierType {
+  private val manager = new CSVReferenceTypeManager("/identifier-schemes.csv")
+
```

Comment:
Should we also have a test for `CSVReferenceTypeManager` for the case where the resource doesn't exist?